pdl-compiler/src/analyzer.rs

```diff
@@ -481,6 +485,10 @@ impl Schema {
         *self.decl_size.get(&key).unwrap()
     }
 
+    pub fn parent_size(&self, key: DeclKey) -> Size {
+        *self.parent_size.get(&key).unwrap()
```

Comment:
I see this is consistent with the other calls, but could you use `self.parent_size[key]` instead to indicate that you're willing to panic if the the key is not found?